src/polymer-language-server.ts

```diff
@@ -1,29 +1,58 @@
-/* --------------------------------------------------------------------------------------------

```

Comment:
what happened here with the whitespaces? LF <-> CRLF?

Reply:
Yeah. Was CRLF, I converted it to just LF

Reply:
👍 :shipit: